lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require "pry"
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
+
+  def score
+    letters = word.split('')
+
+    total = 0
+    letters.each do |letter|
+      total += point_values[letter]
+    end
+    total
+  end
+
   def score(word)
-    1
+    total = 0
+    if word == nil
+      total = nil
```

Comment:
Need to return 0 if the word passed is `nil`! Be sure to follow the interaction pattern.